docs/android/gradle.md

```diff
@@ -1,205 +1,200 @@
 
 # Gradle Build Instructions - Android
 
-These instructions guide you through the process of integrating Product Science instrumentation into an Android application's Gradle build process.
+Follow these steps to instrument your app with PS Plugin. Once you have successfully installed the build and run the app, you will be able to record and analyze traces.
+
+If you get stuck, do refer to our [FAQ](https://docs.productscience.app/android/integration-faq-android/) for additional steps and guidance.
 
 !!! info
```

Comment:
If these instructions are being shared for self-service I would recommend moving the offline build instructions to another file and/or asking to reach out to our sales team. We should avoid handing out an offline version of the plugin to anybody that asks for it